src/Config/ObjectModel/BaseRunTimeConfigLoader.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.DataApiBuilder.Config.Converters;
+using Azure.DataApiBuilder.Config.NamingPolicies;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.DataApiBuilder.Config.ObjectModel
+{
+    public abstract class BaseRunTimeConfigLoader
```

Comment:
```suggestion
    public abstract class RuntimeConfigLoader
```

We should call this `RuntimeConfigLoader` and the current implementation `FileSystemRuntimeConfigLoader` so it's clear what the implementation does.

Reply:
Fixed.